webapp/src/store.js

```diff
@@ -20,7 +20,25 @@ import { etherscan } from 'lib/EtherscanAPI'
 import { rootReducer } from './reducer'
 import { rootSaga } from './sagas'
 
-const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
+const onlyTypesForActions = env.get('REACT_APP_REDUX_ACTION_ONLY_TYPE', false)
+const storeKeys = env.get('REACT_APP_REDUX_STORE_KEYS', '')
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+  ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({
+      actionSanitizer: action =>
+        onlyTypesForActions ? { type: action.type } : action,
+      stateSanitizer: state =>
+        storeKeys.length > 0
+          ? storeKeys.split(',').reduce((sanitizedState, key) => {
+              const value = state[key]
+              if (value) {
+                return { ...sanitizedState, [key]: value }
+              }
+              return sanitizedState
+            }, {})
+          : state
+    })
+  : compose
```

Comment:
Maybe we should do this only when `env.isDevelopment()` ?

Reply:
Hmmm, we can avoid both sanitizers keeping the env empty. Do you think that is still useful to check if the environment is development??